src/HornedBeasts.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react'
+
+export default class HornedBeasts extends Component{
+  render() {
+    return (
+      <div>
+        <h2>{this.props.title}</h2>
+        <img src={this.props.imageURL} atl = 'A picure of a horned beast' ></img>
```

Comment:
Good work so far!

- On line 8, you have a typo for `alt` 
- Feedback so far is to ensure that with attributes on tags, we use syntax of alt="" no space after equal sign 👍